{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"10:31:48 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"10:31:48 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"10:31:48 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"10:31:49 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"10:31:50 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"10:31:50 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"10:31:50 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"10:31:52 AM"}
{"level":"error","message":"HorizonSaleSearch { ss_id: 8067 } { draw: '1',\n  columns: \n   [ { data: 'Action',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'name',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'mobileno',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'channel',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'Telecom_Circle',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'ss_id',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'fba_id',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'group',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'Email',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'Mobile_2',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] },\n     { data: 'SyncDate',\n       name: '',\n       searchable: 'true',\n       orderable: 'false',\n       search: [Object] } ],\n  start: '0',\n  length: '20',\n  search: { value: '', regex: 'false' },\n  ss_id: '8067',\n  session_id: 'NCzQ0VXTcIBTQF47uPPytdUtPCApbULO' }","timestamp":"10:31:52 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:04:29 AM"}
{"level":"error","message":"(node:13592) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): MongoError: E11000 duplicate key error collection: lm-api-qa.sync_contacts index: Sync_Contact_Id_1 dup key: { : null }","timestamp":"11:04:30 AM"}
{"level":"error","message":"(node:13592) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): MongoError: Index with name: Posp_Id_1 already exists with different options","timestamp":"11:04:30 AM"}
{"level":"error","message":"(node:13592) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): MongoError: E11000 duplicate key error collection: lm-api-qa.credentials index: Credential_Id_1 dup key: { : null }","timestamp":"11:04:30 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:46 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:46 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:46 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:46 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:21:46 AM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"11:21:46 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:21:46 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:21:47 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:21:47 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:21:47 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:54 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:54 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:54 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:55 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:55 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:21:56 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:31 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:32 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:32 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:32 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:22:32 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:22:32 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:22:32 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:22:32 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:22:32 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:48 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:48 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:49 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:49 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:50 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:22:50 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:28 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:28 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:28 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:28 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:24:28 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:24:28 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:24:28 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:24:28 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:24:28 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:30 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:31 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:31 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:32 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:24:32 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:25:55 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:25:55 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:25:55 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:25:55 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:25:55 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:25:55 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:25:55 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:25:55 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:25:55 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:01 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:01 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:02 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:02 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:03 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:03 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:04 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:04 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:04 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:26:04 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:26:05 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:26:05 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:26:05 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:26:05 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:06 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:26:06 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:30:49 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:15 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:15 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:15 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:15 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:34:15 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:34:15 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:34:15 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:34:15 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:34:15 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:18 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:28 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:29 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:29 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:30 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:34:43 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:34:43 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:43 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:44 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:44 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:34:45 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:08 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:35:08 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:35:08 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:35:08 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:35:08 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:35:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:10 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:10 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:11 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:11 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:35:12 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:48 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:49 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:49 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:49 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:42:49 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:49 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:42:49 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:42:49 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:42:49 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:42:49 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:49 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:50 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:50 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:42:51 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:15 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:15 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:15 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:15 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:45:15 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:45:15 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:45:16 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:45:16 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:45:16 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:18 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:18 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:19 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:19 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:45:20 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:10 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:10 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:10 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:10 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:10 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:55:19 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:55:19 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:55:19 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:55:19 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:55:19 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:37 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:37 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:37 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:37 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:55:39 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:55:39 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:55:39 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:55:39 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:55:39 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:55:39 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:57:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:57:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:57:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:57:08 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"11:57:11 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:57:11 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:57:11 AM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"11:57:11 AM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"11:57:11 AM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"11:57:11 AM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:34 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:34 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:34 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:34 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:38 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:02:38 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:02:38 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:02:38 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:02:38 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:02:38 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:52 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:53 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:53 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:53 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:02:56 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:02:56 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:02:56 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:02:56 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:02:56 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:02:56 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:08:42 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:08:43 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:08:43 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:08:43 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:08:52 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:08:52 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:08:52 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:08:52 PM"}
{"level":"error","message":"EXCEPTION posps_dsas_view_handler { Ss_Id: 150256,\n  user_type: '',\n  product: 'insurance,loan',\n  status: 'EXCEPTION',\n  channel: '',\n  POSP: 'NA',\n  POSP_USER: null,\n  DOC_LOGS: null,\n  IIB: null,\n  POSP_INSURER: 'NA',\n  EMP: 'NA',\n  RM: null,\n  MAPPING_HISTORY: 'NA',\n  HR: 'NA',\n  SYNC_CONTACT: \n   { ACTION_NEEDED: 'SYNC',\n     FIRST_SYNC_CAMPAIGN_CREATIVE: 'https://origin-cdnh.policyboss.com/fmweb/prod_info/Images/in_miss1.jpeg',\n     RE_SYNC_CAMPAIGN_CREATIVE: 'https://origin-cdnh.policyboss.com/fmweb/prod_info/Images/in_miss1.jpeg' },\n  SYNC_CONTACT_LEAD_PURCHASE: { ACTION_NEEDED: 'PURCHASE' },\n  DEVICE: { ACTION_NEEDED: 'INSTALL' },\n  INSURANCE: { ACTION_NEEDED: 'SALE' },\n  error: 'Error: Can\\'t set headers after they are sent.\\n    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)\\n    at ServerResponse.header (D:\\\\Suyash Files\\\\SourceCode\\\\node_modules\\\\express\\\\lib\\\\response.js:719:10)\\n    at ServerResponse.send (D:\\\\Suyash Files\\\\SourceCode\\\\node_modules\\\\express\\\\lib\\\\response.js:164:12)\\n    at ServerResponse.json (D:\\\\Suyash Files\\\\SourceCode\\\\node_modules\\\\express\\\\lib\\\\response.js:250:15)\\n    at posps_dsas_view_handler (D:\\\\Suyash Files\\\\SourceCode\\\\controllers\\\\posps.js:2464:17)\\n    at D:\\\\Suyash Files\\\\SourceCode\\\\controllers\\\\posps.js:1138:5\\n    at D:\\\\Suyash Files\\\\SourceCode\\\\node_modules\\\\mongoose\\\\lib\\\\query.js:3017:18\\n    at process._tickCallback (internal/process/next_tick.js:103:7)' } Error: Can't set headers after they are sent.\n    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)\n    at ServerResponse.header (D:\\Suyash Files\\SourceCode\\node_modules\\express\\lib\\response.js:719:10)\n    at ServerResponse.send (D:\\Suyash Files\\SourceCode\\node_modules\\express\\lib\\response.js:164:12)\n    at ServerResponse.json (D:\\Suyash Files\\SourceCode\\node_modules\\express\\lib\\response.js:250:15)\n    at posps_dsas_view_handler (D:\\Suyash Files\\SourceCode\\controllers\\posps.js:2464:17)\n    at D:\\Suyash Files\\SourceCode\\controllers\\posps.js:1138:5\n    at D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\query.js:3017:18\n    at process._tickCallback (internal/process/next_tick.js:103:7)","timestamp":"12:08:52 PM"}
{"level":"warn","message":"uncaughtException Error: Can't set headers after they are sent.\n    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)\n    at ServerResponse.header (D:\\Suyash Files\\SourceCode\\node_modules\\express\\lib\\response.js:719:10)\n    at ServerResponse.send (D:\\Suyash Files\\SourceCode\\node_modules\\express\\lib\\response.js:164:12)\n    at ServerResponse.json (D:\\Suyash Files\\SourceCode\\node_modules\\express\\lib\\response.js:250:15)\n    at posps_dsas_view_handler (D:\\Suyash Files\\SourceCode\\controllers\\posps.js:2472:24)\n    at D:\\Suyash Files\\SourceCode\\controllers\\posps.js:1138:5\n    at D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\query.js:3017:18\n    at process._tickCallback (internal/process/next_tick.js:103:7)","timestamp":"12:08:52 PM"}
{"level":"warn","message":"uncaughtException { Error: socket hang up\n    at createHangUpError (_http_client.js:254:15)\n    at Socket.socketOnEnd (_http_client.js:346:23)\n    at emitNone (events.js:91:20)\n    at Socket.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  code: 'ECONNRESET',\n  request: \n   ClientRequest {\n     domain: null,\n     _events: {},\n     _eventsCount: 0,\n     _maxListeners: undefined,\n     href: 'http://localhost:3000/report/sync_emp_master?ss_id=150256',\n     options: \n      { host: 'localhost',\n        port: '3000',\n        path: '/report/sync_emp_master?ss_id=150256',\n        href: 'http://localhost:3000/report/sync_emp_master?ss_id=150256',\n        method: 'GET',\n        headers: [Object] },\n     _httpRequest: \n      Writable {\n        _writableState: [Object],\n        writable: true,\n        domain: null,\n        _events: [Object],\n        _eventsCount: 2,\n        _maxListeners: undefined,\n        _options: [Object],\n        _ended: true,\n        _ending: true,\n        _redirectCount: 0,\n        _redirects: [],\n        _requestBodyLength: 0,\n        _requestBodyBuffers: [],\n        _onNativeResponse: [Function],\n        _currentRequest: [Object],\n        _currentUrl: 'http://localhost:3000/report/sync_emp_master?ss_id=150256' } } }","timestamp":"12:10:52 PM"}
{"level":"warn","message":"uncaughtException { Error: socket hang up\n    at createHangUpError (_http_client.js:254:15)\n    at Socket.socketCloseListener (_http_client.js:286:23)\n    at emitOne (events.js:101:20)\n    at Socket.emit (events.js:188:7)\n    at TCP._handle.close [as _onclose] (net.js:498:12)\n  code: 'ECONNRESET',\n  request: \n   ClientRequest {\n     domain: null,\n     _events: {},\n     _eventsCount: 0,\n     _maxListeners: undefined,\n     href: 'http://localhost:3000/report/sync_emp_master?ss_id=150256',\n     options: \n      { host: 'localhost',\n        port: '3000',\n        path: '/report/sync_emp_master?ss_id=150256',\n        href: 'http://localhost:3000/report/sync_emp_master?ss_id=150256',\n        method: 'GET',\n        headers: [Object] },\n     _httpRequest: \n      Writable {\n        _writableState: [Object],\n        writable: true,\n        domain: null,\n        _events: [Object],\n        _eventsCount: 2,\n        _maxListeners: undefined,\n        _options: [Object],\n        _ended: true,\n        _ending: true,\n        _redirectCount: 0,\n        _redirects: [],\n        _requestBodyLength: 0,\n        _requestBodyBuffers: [],\n        _onNativeResponse: [Function],\n        _currentRequest: [Object],\n        _currentUrl: 'http://localhost:3000/report/sync_emp_master?ss_id=150256' } } }","timestamp":"12:10:52 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:11:30 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:11:30 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:11:30 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:11:30 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:11:34 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:11:34 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:11:34 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:11:34 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:11:34 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:11:34 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:32 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:32 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:32 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:12:32 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:12:32 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:32 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:33 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:33 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:33 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:33 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:37 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:37 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:37 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:12:37 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:12:37 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:37 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:55 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:55 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:55 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:55 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:12:57 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:57 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:57 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:12:57 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:12:57 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:12:57 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:15:47 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:15:47 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:15:47 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:15:47 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:15:49 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:15:49 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:15:49 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:15:49 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:15:49 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:15:49 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:17:59 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:17:59 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:17:59 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:17:59 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:18:01 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:18:01 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:18:01 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:18:01 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:18:01 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:18:01 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:19:47 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:19:48 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:19:48 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:19:48 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:19:50 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:19:50 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:19:50 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:19:50 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:19:50 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:19:50 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:24:21 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:25:12 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:25:12 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:25:12 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:25:17 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:25:17 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:25:17 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:25:17 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:25:17 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:25:17 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:26:16 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:26:43 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:29:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:29:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"12:29:55 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:29:55 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:29:55 PM"}
{"level":"error","message":"DBG-1 8067 4784156","timestamp":"12:29:55 PM"}
{"level":"error","message":"DBG-4 8067 NO_IPM","timestamp":"12:29:55 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::ffff:127.0.0.1 undefined","timestamp":"12:29:55 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:33:28 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:33:28 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:35:57 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:7000/ ::1 string","timestamp":"2:35:57 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:06 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:07 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:07 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:07 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:07 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:07 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:07 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:08 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:08 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:08 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:09 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:09 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:09 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:09 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:09 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:09 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:09 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:09 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:10 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:10 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:10 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:10 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:10 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:11 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:22:11)\n    at Server.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:327:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:317:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Suyash Files\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:187:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"65.0.176.149:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"5:24:11 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:39:25 PM"}
{"level":"error","message":"(node:8756) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): MongoError: E11000 duplicate key error collection: lm-api-qa.sync_contacts index: Sync_Contact_Id_1 dup key: { : null }","timestamp":"5:39:25 PM"}
{"level":"error","message":"(node:8756) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): MongoError: Index with name: Posp_Id_1 already exists with different options","timestamp":"5:39:25 PM"}
{"level":"error","message":"(node:8756) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): MongoError: E11000 duplicate key error collection: lm-api-qa.credentials index: Credential_Id_1 dup key: { : null }","timestamp":"5:39:25 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::1 undefined","timestamp":"5:56:49 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"warn","message":"Db.prototype.authenticate method will no longer be available in the next major release 3.x as MongoDB 3.6 will only allow auth against users in the admin db and will no longer allow multiple credentials on a socket. Please authenticate using MongoClient.connect with auth credentials.","timestamp":"5:59:01 PM"}
{"level":"error","message":"(node:4560) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): MongoError: E11000 duplicate key error collection: lm-api-qa.sync_contacts index: Sync_Contact_Id_1 dup key: { : null }","timestamp":"5:59:02 PM"}
{"level":"error","message":"(node:4560) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): MongoError: Index with name: Posp_Id_1 already exists with different options","timestamp":"5:59:02 PM"}
{"level":"error","message":"(node:4560) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): MongoError: E11000 duplicate key error collection: lm-api-qa.credentials index: Credential_Id_1 dup key: { : null }","timestamp":"5:59:02 PM"}
{"level":"error","message":"LOG REFERER YTD undefined ::1 undefined","timestamp":"5:59:08 PM"}
{"level":"error","message":"SSID DONE -  150009","timestamp":"5:59:08 PM"}
